.travis.yml

```diff
@@ -1,23 +1,25 @@
 language: clojure
 lein: lein2
+install: ./ext/travisci/install.sh ~/.lein/profiles.clj
```

Comment:
I think it would be best to not take any changes from the CVE fork into this file.  A lot of the info was specific to the CVE fork.